src/cpp/jank/runtime/behavior/callable.cpp

```diff
@@ -1,3 +1,4 @@
+#include "jank/runtime/obj/persistent_vector.hpp"
```

Comment:
Check if you need this. If you do, please put it after the standard includes and use <>.